2-mandatory/3-financial-times.js

```diff
@@ -24,6 +37,14 @@ function titleWithFewestWords(allArticleTitles) {
 */
 function headlinesWithNumbers(allArticleTitles) {
     // TODO
+    let numArticle = [];
+    let pattern =/[0-9]/g;
```

Comment:
Do you know what the `g` suffix does? How will it affect the result of the search? (this all works by the way, I'm just asking ;)